Standardize error messages in PR skill prerequisite checks

## Context

The PR skills (`/pr:get-details`, `/pr:update`, `/pr:fix-comments`) now include prerequisite checks that verify a PR exists before executing. These checks use emoji in their output:

```bash
if [ -s /tmp/.pr_status ]; then 
    echo "✓ PR exists: $(cat /tmp/.pr_status)"
else 
    echo "✗ No PR created yet"
fi
```

And in error messages:
```bash
echo "❌ Error: No PR has been created yet."
```

## Issue

Using emoji (✓/✗/❌) may cause display issues in some terminal environments that don't support Unicode properly, leading to:
- Garbled characters
- Missing symbols
- Inconsistent rendering across different terminals

## Suggestion

Consider standardizing on one approach:

**Option 1: Text-only**
```bash
echo "[OK] PR exists: $(cat /tmp/.pr_status)"
echo "[WARN] No PR created yet"
echo "[ERROR] No PR has been created yet."
```

**Option 2: Keep emoji but add fallback**
```bash
# Detect terminal capability and use appropriate format
```

**Option 3: Make it configurable**
Allow users to set a preference in settings

## Affected Files

- `plugins/pr/skills/get-details/SKILL.md`
- `plugins/pr/skills/update/SKILL.md`
- `plugins/pr/skills/fix-comments/SKILL.md`

## Priority

Low - This is a nice-to-have improvement for better terminal compatibility, not a critical bug.

--- a/plugins/pr/skills/fix-comments/SKILL.md
+++ b/plugins/pr/skills/fix-comments/SKILL.md
@@ -20,12 +20,12 @@ Automatically address feedback from GitHub pull request comments.
 ## Prerequisites
 
 **Check PR Status:**
-!`if [ -s /tmp/.pr_status ]; then echo "✓ PR exists: $(cat /tmp/.pr_status)"; else echo "✗ No PR created yet"; fi`
+!`if [ -s /tmp/.pr_status ]; then echo "[OK] PR exists: $(cat /tmp/.pr_status)"; else echo "[WARN] No PR created yet"; fi`
 
 **Before proceeding, verify PR exists:**
 ```bash
 if [ ! -s /tmp/.pr_status ]; then
-    echo "❌ Error: No PR has been created yet."
+    echo "[ERROR] No PR has been created yet."
     exit 1
 fi
 ```

--- a/plugins/pr/skills/get-details/SKILL.md
+++ b/plugins/pr/skills/get-details/SKILL.md
@@ -10,12 +10,12 @@ Retrieves and displays pull request information including title, description, ch
 ## Prerequisites
 
 **Check PR Status:**
-!`if [ -s /tmp/.pr_status ]; then echo "✓ PR exists: $(cat /tmp/.pr_status)"; else echo "✗ No PR created yet"; fi`
+!`if [ -s /tmp/.pr_status ]; then echo "[OK] PR exists: $(cat /tmp/.pr_status)"; else echo "[WARN] No PR created yet"; fi`
 
 **Before proceeding, verify PR exists:**
 ```bash
 if [ ! -s /tmp/.pr_status ]; then
-    echo "❌ Error: No PR has been created yet."
+    echo "[ERROR] No PR has been created yet."
     exit 1
 fi
 ```

--- a/plugins/pr/skills/update/SKILL.md
+++ b/plugins/pr/skills/update/SKILL.md
@@ -27,12 +27,12 @@ $ARGUMENTS
 ## Prerequisites
 
 **Check PR Status:**
-!`if [ -s /tmp/.pr_status ]; then echo "✓ PR exists: $(cat /tmp/.pr_status)"; else echo "✗ No PR created yet"; fi`
+!`if [ -s /tmp/.pr_status ]; then echo "[OK] PR exists: $(cat /tmp/.pr_status)"; else echo "[WARN] No PR created yet"; fi`
 
 **Before proceeding, verify PR exists:**
 ```bash
 if [ ! -s /tmp/.pr_status ]; then
-    echo "❌ Error: No PR has been created yet."
+    echo "[ERROR] No PR has been created yet."
     exit 1
 fi
 ```

